web/app/controller/Root.js

```diff
@@ -124,7 +124,15 @@ Ext.define('Traccar.controller.Root', {
         if (Traccar.app.isMobile()) {
             Ext.create('widget.mainMobile');
         } else {
-            Ext.create('widget.main');
+            main = Ext.create('widget.main');
+            if (Traccar.app.getAttributePreference('web.disableReport', false)) {
```

Comment:
The best way is `main.lookupReference('reportView')` but unfortunately it does not work (return `null`) here is similar problem without solution https://www.sencha.com/forum/showthread.php?330708-Event-in-child-controller-damages-lookupReference-called-in-parent-controller

We can use `main.items.items[5]` directly but I'm afraid we can forget this and change layout.
Also this loop might be reused to hide Events panel if we implement it in future.